README.md

```diff
@@ -608,14 +608,23 @@ IEEE Conf. Computer Vision and Pattern Recognition Workshops (CVPRW), 2021. [You
 - <a name="Chui21arxiv"></a>Chui, J., Klenk, S., Cremers, D.,  
 *[Event-Based Feature Tracking in Continuous Time with Sliding Window Optimization](https://arxiv.org/pdf/2107.04536)*,  
 arXiv preprint arXiv, 2021.
+- <a name="glover21tpami"></a>Glover A, Dinale A, De Souza Rosa L, Bamford S, Bartolozzi C   
+*[luvharris: A practical corner detector for event-cameras](https://doi.org/10.1109/TPAMI.2021.3135635)*,  
+TPAMI, 2021. [Code](https://github.com/robotology/event-driven)
```

Comment:
Thanks. Could you please format the publication venues like others so that they all have the same format?

This comment applies to all entries added in this PR.

Reply:
yes sure! done